src/components/CustomerDetails/CustomerDetails.test.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import {mount} from 'enzyme';
+import CustomerDetails from '../CustomerDetails';
+
+const wrapper = mount(
```

Comment:
I'd wrap everything in a `describe` block

and do this mount in a `beforeEach` block

e.g.

```es6
describe('something', () => {
  let someVariable;
  beforeEach(() => {
    someVariable = new Thing();
  });

  it('does a thing', () => {
    expect(someVariable.magic()).toEqual('magic');
  });
})
```